### ISSUE TITLE
Allow adding prompts for selection with fish_config tool

### ISSUE THREAD

**Issue by cappslock** [fish-shell/fish-shell#2577]
The fish_config tool is useful for making adjustments to the color scheme and prompt. It would be nice if there was an obvious way to add a prompt or color scheme to the available options in this tool.

I do understand how to change both manually, but being able to do so from this tool would be nice.

**Comment by ridiculousfish** [comment]
Do you mean add one of your own directly from the tool? Or did you have in mind something like an online repository?

**Comment by cappslock** [comment]
At the most basic level, I mean to provide some way in which user themes can be made available for selection from the tool. Sure, something like an online repo would be cool, but that seems like more of a user-land project. I think just maybe a directory or variable or some other way in which themes could be made available for preview and selection within the tool would be cool. Adding from a local file within the tool itself would also be cool.

**Comment by ridiculousfish** [comment]
Which version of fish are you using? fish 2.2.0 has themes available for preview and selection:

![screen shot 2015-11-30 at 1 09 29 pm](https://cloud.githubusercontent.com/assets/920838/11484581/a2a398e4-9763-11e5-84cd-74687775715a.png)

is that different from what you see?

**Comment by cappslock** [comment]
This is the same as what I see, I'm using 2.2. What I mean is I would like some way to add new selections in this tool. Basically I want to be able to download some user created content and stick it in a directory and have it magically show up here, or tell the config tool a repo location or something like that and have that prompt/color scheme available.

**Comment by ridiculousfish** [comment]
Gotcha, thanks.

**Comment by tsteinholz** [comment]
I agree, and to add on, fish should have some sort of customization for prompts similar to sites like http://bashrcgenerator.com/ which allow you to create your own prompt instead of just using one of the pre-defines. Another thing lacking is customization in the prompt colors. Event further down the road, if a tool like http://terminal.sexy could be implemented into the interface would be amazing :100:
